sdk/storage/Azure.Storage.Common/src/Shared/TransferValidationOptionsExtensions.cs

```diff
@@ -9,7 +9,11 @@ public static ValidationAlgorithm ResolveAuto(this ValidationAlgorithm validatio
         {
             if (validationAlgorithm == ValidationAlgorithm.Auto)
             {
+#if BlobSDK || DataLakeSDK
                 return ValidationAlgorithm.StorageCrc64;
+#elif FileSDK // file shares don't support crc64
+                return ValidationAlgorithm.MD5;
+#endif
```

Comment:
Should probably include `#else throw` in here for safety

Reply:
> @jaschrep-msft, can you update the Files changelog?

@seanmcc-msft The bugged code hasn't actually been released yet. There would be no change to alert someone to.